src/MissionManager/PlanMasterController.h

```diff
@@ -155,5 +186,7 @@ private slots:
     QmlObjectListModel*     _planCreators =             nullptr;
     DataClass* _dataClass;
     QString m_url;
+    int m_planIndex = 0;
```

Comment:
How many plans are we going to be holding? Sure int is enough right?

Reply:
Might make sense to have a larger data type if you need, or even with int, have a max bound for how many plans one is allowed to create/keep, and have a check to either delete older plans or overwrite in a circular fashion or alteast alert the operator.

@hrishiballal @rhythmize

Reply:
I'd suggest to atleast make it `unsigned int`. It'll double the current limit and then we can have some checks to prevent any rollovers.

Reply:
Changed it to unsigned long long int